pkg/timeline/service/push.ts

```diff
@@ -16,8 +19,74 @@ export class PushTimelineService {
     private readonly noteVisibility: NoteVisibilityService,
     private readonly timelineNotesCacheRepository: TimelineNotesCacheRepository,
     private readonly fetchSubscribedListService: FetchSubscribedListService,
+    /**
+     * @description Limit of timeline cache
```

Comment:
```suggestion
     * @description Limit length of timeline caches
```
単位が不明瞭なので, キャッシュのエントリ数であることを説明に書いてほしいです.